File: Frontend/src/app/models/responses/LoginResponse.ts
import { Token } from "../entities/Token";
import { User } from "../entities/User";

export class LoginResponse
{
    private _token: Token;
    private _user: User;
    private message: string;

    constructor(token: Token, user: User, $message: string)
    {
        this._token = token;
        this._user = user;
        this.message = $message;
    }

    public get token(): Token
    {
        return this._token;
    }

    public get user(): User
    {
        return this._user;
    }

    public get $message(): string
    {
        return this.message;
    }

    public set token(value: Token)
    {
        this._token = value;
    }

    public set user(value: User)
    {
        this._user = value;
    }

    public set $message(value: string)
    {
        this.message = value;
    }
}

File: Backend/src/main/java/hadi/springSecurity/models/responses/LoginResponse.java
package hadi.springSecurity.models.responses;

import hadi.springSecurity.models.entities.Token;
import hadi.springSecurity.models.entities.User;

public class LoginResponse
{
	private Token token;
	private User user;
	private String message;

	public LoginResponse()
	{
		super();
	}

	public LoginResponse(Token token, User user, String message)
	{
		super();
		this.token = token;
		this.user = user;
		this.message = message;
	}

	public Token getToken()
	{
		return token;
	}

	public void setToken(Token token)
	{
		this.token = token;
	}

	public String getMessage()
	{
		return message;
	}

	public void setMessage(String message)
	{
		this.message = message;
	}

	public User getUser()
	{
		return user;
	}

	public void setUser(User user)
	{
		this.user = user;
	}	
}
